File: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples-js/dedicatedHostExamples/DedicatedHostGroups_Delete_MaximumSet_Gen.js
const { ComputeManagementClient } = require("@azure/arm-compute");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Delete a dedicated host group.
 *
 * @summary Delete a dedicated host group.
 * x-ms-original-file: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/dedicatedHostExamples/DedicatedHostGroups_Delete_MaximumSet_Gen.json
 */
async function dedicatedHostGroupsDeleteMaximumSetGen() {
  const subscriptionId = "{subscription-id}";
  const resourceGroupName = "rgcompute";
  const hostGroupName = "a";
  const credential = new DefaultAzureCredential();
  const client = new ComputeManagementClient(credential, subscriptionId);
  const result = await client.dedicatedHostGroups.delete(resourceGroupName, hostGroupName);
  console.log(result);
}

dedicatedHostGroupsDeleteMaximumSetGen().catch(console.error);

File: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-03-01/examples-java/dedicatedHostExamples/DedicatedHostGroups_Delete_MaximumSet_Gen.java
import com.azure.core.util.Context;

/** Samples for DedicatedHostGroups Delete. */
public final class Main {
    /*
     * x-ms-original-file: specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/examples/dedicatedHostExamples/DedicatedHostGroups_Delete_MaximumSet_Gen.json
     */
    /**
     * Sample code: DedicatedHostGroups_Delete_MaximumSet_Gen.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void dedicatedHostGroupsDeleteMaximumSetGen(com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .virtualMachines()
            .manager()
            .serviceClient()
            .getDedicatedHostGroups()
            .deleteWithResponse("rgcompute", "a", Context.NONE);
    }
}
